src/components/VenueList/VenueCard.jsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Card, Typography } from '@mui/material';
+import { LocationOn, People, Star } from '@mui/icons-material';
+import { Link } from 'react-router-dom';
+
+export const VenueCard = ({ venue }) => (
+  <Card
+    sx={{
+      transition: 'transform 0.2s ease-in-out, box-shadow 0.2s ease-in-out',
+      '&:hover': {
+        transform: 'translateY(-4px)',
+        boxShadow: '0 8px 25px rgba(0,0,0,0.15)',
+      },
+      cursor: 'pointer',
+    }}
+    elevation={0}
+    component={Link}
+    to={`/venue-details/${venue.id}`}
+  >
+    <Box
+      sx={{
+        height: 150,
+        background: `linear-gradient(45deg, #${Math.floor(Math.random() * 16777215).toString(16)}66, #${Math.floor(Math.random() * 16777215).toString(16)}66)`,
+        position: 'relative',
+      }}
+    >
+      <Box
+        sx={{
+          position: 'absolute',
+          top: 0,
+          right: 0,
+          padding: 1,
+          backgroundColor: 'rgba(0,0,0,0.7)',
+          borderBottomLeftRadius: 12,
+        }}
+      >
+        <Typography
+          sx={{
+            fontWeight: 600,
+            textTransform: 'capitalize',
+            color: 'white',
+            fontSize: 14,
+          }}
+        >
+          {venue.name}
+        </Typography>
+      </Box>
+      <Box
+        sx={{
+          position: 'absolute',
+          bottom: 0,
+          left: 0,
+          padding: 1,
+          backgroundColor: 'rgba(0,0,0,0.7)',
+          borderTopRightRadius: 12,
+        }}
+      >
+        <Typography
+          sx={{ fontSize: 14, color: 'white', fontWeight: 600 }}
+        >{`€${venue.pricePerNightInEUR.toFixed(0)} / night`}</Typography>
```

Comment:
This code is rather difficult to read. It would be better if there were smaller components and separate variables.